### DISCUSSION TITLE
Bug Fixes

### DISCUSSION THREAD

**Discussion by kobrien02** [bcholmes/StarTrek2d20#224] [General]
I was just playing around with the character creator and came across 2 instances where you get stuck and cannot continue. They are both when you select an 'Allied Military' character. One is when Jem'Hadar is selected and the other is the Reman option. Both of these are requiring you to select a talent before continuing, but there are no talents displayed to choose from. Wanted to see if this was in error.

**Comment by bcholmes** [comment]
Should be fixed in the latest version.
